src/common/servo.rs

```diff
@@ -0,0 +1,39 @@
+use std::sync::{Arc, Mutex};
+
+use super::{actuator::Actuator, status::Status};
+
+pub static COMPONENT_NAME: &str = "motor";
+
+pub trait Servo: Status + Actuator {
+    /// Moves the servo to an angular position of `angle_deg` away
+    /// from the home position
```

Comment:
Is there an (in)valid range?

Reply:
this is driver dependent, but I'll add a comment on the gpio servo implementation of the method to explain the clamping behavior